homeassistant/components/ecoal_boiler/__init__.py

```diff
@@ -0,0 +1,61 @@
+"""
+Support for ecoal/esterownik.pl coal/wood boiler controller.
+
+Allows read various readings available in controller
+and set very basic switches.
+"""
```

Comment:
Please refer to other component about the layout of the file header.

Reply:
Yes, but say it's a component for the component.

Reply:
Is it OK in current version?